platform-hub-api/app/jobs/privileged_token_expirer_job.rb

```diff
@@ -0,0 +1,50 @@
+class PrivilegedTokenExpirerJob < ApplicationJob
+  queue_as :tokens_expire
```

Comment:
Just to keep consistent, could you make this `tokens_expire` (and then update the `is_already_queued` method below too)?